src/actions/account.js

```diff
@@ -229,27 +229,22 @@ export const login = ({ passphrase, publicKey, hwInfo }) => async (dispatch, get
   }).then(async (accountData) => {
     const expireTime = (passphrase && getState().settings.autoLog) ?
       Date.now() + accountConfig.lockDuration : 0;
+    let btcAccountData;
+    if (getState().settings.token.list.BTC) {
```

Comment:
Is it possible to merge this with with the other `getState` call on this function, something like `const { network: networkConfig, settings } = getState()`, so there is no need to call it again?